bin/add_celltypes_to_sce.R

```diff
@@ -216,6 +226,17 @@ if (!is.null(opt$cellassign_predictions)) {
   # note that if `metadata(sce)$celltype_methods` doesn't exist yet, this will
   #  come out to just the string "cellassign"
   metadata(sce)$celltype_methods <- c(metadata(sce)$celltype_methods, "cellassign")
+
+  # add cellassign reference organs to metadata
+  cellassign_organs <- opt$celltype_ref_metafile |>
+    readr::read_tsv() |>
+    dplyr::filter(celltype_ref_name == cellassign_ref_info[["ref_name"]]) |>
+    dplyr::pull(organs)
+
+  if (cellassign_organs == "" | is.na(cellassign_organs)) {
```

Comment:
Truthiness is fun! (And Groovy treats NA as falsey)
```suggestion
  if (!cellassign_organs) {
```